app/models/child_object.rb

```diff
@@ -96,9 +96,21 @@ def copy_to_access_master_pairtree
   # rubocop:enable  Metrics/MethodLength
   # rubocop:enable  Layout/LineLength
 
-  def access_master_checksum_matches?
-    access_master_checksum = Digest::SHA1.file(access_master_path).to_s
-    checksum == access_master_checksum
+  def checksum_matches?
+    access_sha256_checksum = Digest::SHA256.file(access_master_path).to_s
+    access_sha1_checksum = Digest::SHA1.file(access_master_path).to_s
+    access_md5_checksum = Digest::MD5.file(access_master_path).to_s
+    if checksum.present?
```

Comment:
Curious on the order.  I was thinking it'd be sha512 (preservica or manually updated by user), checksum (aka sha1 from goobi), sha256 (lb), md5 (lb).

I would put preservica and goobi checksums first since they should always be right unless something is wrong with the file. The ladybird checksums will be wrong in a lot of cases.
Any may be updated from preservica at some point, so the sh512 is the most likely to be right.